### PR TITLE
Chloe create more

### PR DESCRIPTION
**Pull request by chloemackallah**

### CODE REVIEW

**Review by hot007 — APPROVED** [review]
This looks good to me but maybe @organisergirl could review the DMP section?

File: Governance/create/create-basics.md [hunk 1]
```diff
@@ -15,24 +15,32 @@ NetCDF files contain three main components:
...
 
 It is likely that the raw scientific data that you are building your datasets around are already highly structured (even in netCDF format already), so your main effort here will be ensuring that the metadata correctly and adequately describes your data. 
 
 ### Attributes
 
-NetCDF attributes are in one of two categories: Machine-readable metadata attributes and human-readable metadata attributes (long names, most global attrs).
+NetCDF metadata attributes are generally in one of two categories: machine-readable and human-readable (though these overlap significantly).
 
 * Machine-readable attributes (e.g., `units`, `standard_name`, `missing_value` and `calendar`) typically describe the data itself, are usually variable-level, and can be automatically interpreted by standard plotting and analysis tools if set according to common [conventions](../concepts/conventions.md) and [controlled vocabularies](../concepts/controlled-vocab.md). These conventions typically contain suggestions for the attribute key and value using commonly understood terms, and are highly recommended to enable analysis and visualisation with standard software packages.
```
Comment (paolap) [inline Governance/create/create-basics.md:33]: Doesn't need to happen before merging this but I think linking to the tech side of this https://acdguide.github.io/Governance/tech/conventions.html would be better as this covers in details all the aspects of these attributes (both variables and globals)

Comment (paolap) [reply Governance/create/create-basics.md:33]: sorry just so this happens later so I'll resolve this myself

**Review by paolap — APPROVED** [review]

**Review by organisergirl — APPROVED** [review]
Looks good to me.

**Pull request merged**